File: net/loveruby/cflat/util/ErrorHandler.java
package net.loveruby.cflat.util;

import java.io.OutputStream;
import java.io.PrintStream;

public class ErrorHandler {
    protected String programId;
    protected PrintStream stream;
    protected long errCnt;
    protected long warnCnt;

    public ErrorHandler(String programId) {
        programId = programId;
        stream = System.err;
    }

    public ErrorHandler(String programId, OutputStream stream) {
        programId = programId;
        this.stream = new PrintStream(stream);
    }
}

File: app/src/main/kotlin/com/ksys/cflat/util/ErrorHandler.kt
package com.ksys.cflat.util

import java.io.OutputStream
import java.io.PrintStream

class ErrorHandler {
    val programId: String
    var stream: PrintStream
    // var errCnt: long
    // var warnCnt: long

    constructor(programId: String) {
        this.programId = programId
        stream = System.err
    }

    constructor(programId: String, stream: OutputStream) {
        this.programId = programId
        this.stream = PrintStream(stream)
    }
}
